src/index.html

```diff
@@ -10,8 +10,9 @@
     <div class="wrapper">
       <div class="container-calendar">
         <div id="right">
-          <button id="create">Create</button>
+          <button id="create" onclick="window.location.href = 'journal.html'">Create</button>
```

Comment:
please use a js file and attach an onclick event instead of embedding javascript into the html